packages/vm/src/wasm_backend/gatekeeper.rs

```diff
@@ -34,6 +34,10 @@ struct GatekeeperConfig {
     /// Note, this feature is not yet standardized!
     /// See <https://webassembly.org/roadmap/> and <https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md>.
     allow_feature_threads: bool,
+    /// True iff operations of the "Shared-Everything Threads" feature are allowed.
```

Comment:
```suggestion
    /// True if operations of the "Shared-Everything Threads" feature are allowed.
```

Reply:
Alright, okay. I've never seen it written like that before personally but if it's intentional then that's okay

Reply:
The abbreviation is a bit math nerd specific, but it made it from math into computer science as well. But not something I have seen much outside of academia.